TicketSystem/TicketSystem/Data/ApplicationContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.SqlServer;
+using TicketSystem.Models;
+
+namespace TicketSystem.Data
+{
+    public class ApplicationContext : DbContext
+    {
+        public DbSet<User> Users { get; set; }
+        public DbSet<Ticket> Tickets { get; set; }
+        public DbSet<Message> Messages { get; set; }
+
+        public ApplicationContext()
+        {
+            Database.EnsureCreated();
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlServer("Server=(localdb)\\mssqllocaldb;Database=ticketsystem;Trusted_Connection=True;");
```

Comment:
Use normal sql server, not localdb

Reply:
At this stage this is not particularly necessary